config/laravel-mailable-test.php

```diff
@@ -7,5 +7,5 @@
      * of a mailable. This can be any class as long as it
      * extends \Spatie\MailableTest\ArgumentValueProvider::class
      */
-    'argument_value_provider_class' => \Spatie\MailableTest\ArgumentValueProvider::class,
+    'argument_value_provider_class' => \Spatie\MailableTest\FakerArgumentValueProvider::class,
```

Comment:
Something's wrong in my PR atm with the `ArgumentValueProvider` bindings. Maybe dump the config value, and if people want to use their own they need to register the interface in the container? Or too mush of a hassle?